### ISSUE TITLE
Add WEIGHT_SPECTRUM to known MS schema

### ISSUE THREAD

**Issue by Mulan-94** [ska-sa/dask-ms#55]
* Dask-ms 0.2.0rc2:
* Python 3.6.8
* Ubuntu 16.04

Daskms fails on MS with `WEIGHT_SPECTRUM` column
I concluded this after printing `column` at this juncture:
 https://github.com/ska-sa/dask-ms/blob/1587bf7b1251a19e33353dd6faa9e7149ea14179/daskms/reads.py#L210
```
Traceback (most recent call last):
  File "/home/andati/test_vis/gridplots/sources/bokeh/ragavi/ragavi/visibilities.py", line 745, in get_ms
    if data_col not in ['DATA', 'CORRECTED_DATA']:
  File "/home/andati/virtual_environments/ragavi3/lib/python3.6/site-packages/daskms/dask_ms.py", line 291, in xds_from_ms
    **kwargs)
  File "/home/andati/virtual_environments/ragavi3/lib/python3.6/site-packages/daskms/dask_ms.py", line 229, in xds_from_table
    **kwargs).datasets()
  File "/home/andati/virtual_environments/ragavi3/lib/python3.6/site-packages/daskms/reads.py", line 367, in datasets
    return [self._single_dataset(orders)]
  File "/home/andati/virtual_environments/ragavi3/lib/python3.6/site-packages/daskms/reads.py", line 297, in _single_dataset
    short_table_name(self.table))
  File "/home/andati/virtual_environments/ragavi3/lib/python3.6/site-packages/daskms/reads.py", line 213, in _dataset_variable_factory
    chunks, exemplar_row)
  File "/home/andati/virtual_environments/ragavi3/lib/python3.6/site-packages/daskms/columns.py", line 210, in column_metadata
    dims = table_schema[column]['dims']
TypeError: 'set' object is not subscriptable
```

**Comment by sjperkins** [comment]
Thanks, this was a miss by me. 

Should be fixed by https://github.com/ska-sa/dask-ms/pull/56

**Comment by sjperkins** [comment]
Closed by #56

**Comment by Mulan-94** [comment]
> Thanks, this was a miss by me.
> 
> Should be fixed by #56

Thanks !
